internal/cli/cmd/dashboard/dashboard.go

```diff
@@ -16,9 +17,13 @@ func NewCommand(cfg config.CLIConfig) *cobra.Command {
 	cmdOpts := &dashboardOptions{Config: cfg}
 
 	return &cobra.Command{
-		Use:     "dashboard",
-		Short:   "Open the Kargo Dashboard in your default browser.",
-		Example: "kargo logout",
+		Use:   "dashboard",
+		Short: "Open the Kargo Dashboard in your default browser",
+		Args:  option.NoArgs,
+		Example: `
+# Open the Kargo Dashboard
+kargo dashboard
+`,
```

Comment:
Do you think we should specify that users will have to log in to the dashboard separately from the UI?

I'm torn. I feel like it's like "Caution. Coffee may be hot." Like I think that _shouldn't_ be that surprising, but I can somehow still imagine someone opening an issue about it.

Reply:
Reading this again, I was personally more thinking it would require "[...] in the default browser" as a comment 😅 

I think users should not be surprised about having to login again.

Reply:
I was already torn. I'm fine with whatever side of the fence you landed on.